### PR TITLE
Use RPC health check instead of `/health` for `is_connected`

### PR DESCRIPTION
**Pull request by mp-freatic**
Fixing #119

### CODE REVIEW

**Comment by mp-freatic** [comment]
Hey @michaelhly , I don't seem to be able to run the integration tests locally, but unit tests are passing, and I'm not sure what I'm missing there. They all fail with `SolanaRpcException`

image

Happy to attempt to if I can get more info on how to run this 🙂

**Review by michaelhly — APPROVED** [review]
LGTM!

**Comment by michaelhly** [comment]
Actually, @mp-freatic. We need to put back some implementation of `is_connected` for the integration tests:
https://github.com/michaelhly/solana-py/blob/890cbbb8581c5d21496b5936d6218413bda27280/tests/conftest.py#L127

File: src/solana/rpc/providers/async_base.py [hunk 1]
```diff
@@ -12,7 +12,3 @@ class AsyncBaseProvider:
     async def make_request(self, body: Body, parser: Type[T]) -> T:
         """Make a request ot the rpc endpoint."""
         raise NotImplementedError("Providers must implement this method")
-
-    async def is_connected(self) -> bool:
-        """Health check."""
-        raise NotImplementedError("Providers must implement this method")
```
Comment (michaelhly) [inline src/solana/rpc/providers/async_base.py:18]: Can we put this back?

**Comment by mp-freatic** [comment]
🎉 Awesome 🤗

**Pull request merged**